src/annotator/plugin/test/pdf-test.js

```diff
@@ -215,4 +215,101 @@ describe('annotator/plugin/pdf', () => {
       assert.notCalled(fakeAnnotator.anchor);
     });
   });
+
+  describe('#contentContainer', () => {
+    let container;
+    afterEach(() => {
+      container?.remove();
```

Comment:
Not currently, but because the `container` initialization is not in a `beforeEach` block it is not "guaranteed" to be non-nullish. The decision to put the `container` initialization in a test vs. in a `beforeEach` block was slightly arbitrary here. It might make sense to do this if new tests are added in future.